buildconf

```diff
@@ -117,8 +126,18 @@ fi
 need_autoconf="2.57"
 ac_version=`${AUTOCONF:-autoconf} --version 2>/dev/null|head -n 1| sed -e 's/^[^0-9]*//' -e 's/[a-z]* *$//'`
 if test -z "$ac_version"; then
-  if [[ "$OSTYPE" == "darwin"* ]]; then
-    brew install autoconf
+  if [[ "$OSTYPE" == "darwin"* ]]; then   
+      read -p "Do you wish to install autoconf using Homebrew ? y/n `echo $'\n> '`" yn
```

Comment:
How about instead giving buildconf a command line argument like `--install` that would give it the permission to install missing components as then you don't have to add so much duplicated logic in several places in this script.

It could then be extended for other distros too in a future.